Runigram.java

```diff
@@ -101,7 +165,19 @@ public static Color[][] grayScaled(Color[][] image) {
 	 */
 	public static Color[][] scaled(Color[][] image, int width, int height) {
 		//// Replace the following statement with your code
-		return null;
+		Color[][] newimage = new Color[height][width];
+		int newwidth ;
+		int newheight;
+		for (int i=0; i< height; i++)
+		{
+			newheight = (int) (i* (image.length)/height);
+			for(int j=0; j< width; j++)
+			{
+				newwidth =(int) (j * (image[0].length)/width);
+				newimage[i][j] = image[ newheight][ newwidth] ;

```

Comment:
image[newHeight][newWidth] more common spacing. Or if needed image[ newHeight ][ newWidth ] 
The variables itself is better to call in camelCase: newWidth instead of newwidth